### ISSUE TITLE
support readPreference/readPreferenceTags ?

### ISSUE THREAD

**Issue by bardouni** [denodrivers/mongo#377]
Looks like readPreference and readPreferenceTags options are totally ignored
Is there any workaround for it ?

**Comment by nathanclevenger** [comment]
+1 this is also something we're struggling with.  @bardouni did you find any workaround?

**Comment by erfanium** [comment]
@nathanclevenger maybe you can try import from `npm:mongodb`

**Comment by nathanclevenger** [comment]
Thanks @erfanium ... I can use `npm:mongodb` for self-hosted mongo, but we're currently using Deno deploy and it's not supported ... I might have to try spinning up Deno containers on Cloud Run so we can use `npm:mongodb`

**Comment by lucsoft** [comment]
Can't you use a cdn like esm.sh?

**Comment by bardouni** [comment]
> +1 this is also something we're struggling with. @bardouni did you find any workaround?

Not really, I was just experimenting at that time to see if this would work for new ideas
